spotitude.py

```diff
@@ -39,3 +40,36 @@
 
         # Create visualization
         make_visualization(top_tracks_df)
+
+        eel.init(".")
+
+        @eel.expose
+        def create_playlist():
+            """
+                Handles button press from top tracks visualization
+            """
+            spotitude_playlist = Playlist(spotify)
+            spotitude_playlist.create_spotitude_playlist(
+                args.time_range, top_tracks_df["id"].tolist()
+            )
+            print(
+                f"Playlist '{spotitude_playlist.name}' created.\n{spotitude_playlist.url}"
+            )
+
+            # Change 'Create Playlist' to 'Open Playlist' and write hidden URL to html file
+            with open("index.html", "r") as spot_html:
+                filedata = spot_html.read()
+                filedata = filedata.replace("Create Playlist", "Open Playlist")
+                # Change onclick action to open playlist url, also possible independent from python script
+                filedata = filedata.replace(
+                    "onclick='create_playlist()'",
+                    f"onclick=\"window.open('{spotitude_playlist.url}','_blank','resizable=yes')",
+                )
+
+            with open("index.html", "w") as spot_html:
+                spot_html.write(filedata)
+
+            # pylint: disable=no-member
+            eel.open_url(spotitude_playlist.url)  # open playlist in web browser
+
+        eel.start("index.html", mode="chrome")
```

Comment:
Will this line still work if the user didn't have Chrome installed?

Reply:
According to the source code of Eel, specifically this [file](https://github.com/ChrisKnott/Eel/blob/master/eel/browsers.py), it should open the URL with the default browser as a fallback. However, this means that it can't open in "app-mode" meaning that there is no search bar etc.